src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -10676,6 +10676,16 @@ public async Task FormatAttributeOnLambdaInInvocation()
 ");
         }
 
+        [Fact]
+        public async Task FormatAttributeOnLambdaParameter()
+        {
+            await AssertFormatAsync(expected: """
+                var f = ([Attribute] int x = 1) => x;
```

Comment:
In addition to testing the IDE formatter, we should also cover the compiler `NormalizeWhitespace` if not done already.

Reply:
Thanks, added: 2ef39d291d5e1ee1afb8aa3531eb331e6baa8fbe